### PR TITLE
update brew command for kn

### PR DESCRIPTION
**Pull request by sebgoa**
a formulae was merged in homebrew core to install kn without a specific tap

### CODE REVIEW

File: docs/snippets/install-kn.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
         For macOS, you can install `kn` by using [Homebrew](https://github.com/knative/homebrew-client){target=_blank}.
```
Comment (antoineco) [inline docs/snippets/install-kn.md:4]: Is this still relevant?

Comment (sebgoa) [reply docs/snippets/install-kn.md:4]: I created an issue in the homebrew-client repo https://github.com/knative/homebrew-client/issues/32 and I will updated this PR.

you are correct we don't need this homebrew client repo

Comment (sebgoa) [inline docs/snippets/install-kn.md:4]:
```suggestion
        For macOS, you can install `kn` by using [Homebrew](https://brew.sh/){target=_blank}.
```

Comment (sebgoa) [reply docs/snippets/install-kn.md]: I replaced the URL with the one of brew main page.

**Review by antoineco — APPROVED** [review]

**Comment by sebgoa** [comment]
I solved the conflict

**Review by antoineco — APPROVED** [review]

**Comment by omerbensaadon** [comment]
thanks @sebgoa!!

**Pull request merged**